marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mapping/getReferences.api

```diff
@@ -0,0 +1,14 @@
+{
+    "functionName": "getReferences",
+    "desc": "Gets the references  associated with given mapping step name.",
+    "params": [
+        {
+           "name": "stepName",
+           "datatype": "string"
+        }
+    ],
+    "return": {
+        "datatype": "jsonDocument",
```

Comment:
Let's add a description here so it's clear what's returned - i.e. an array of objects, where each object has a name and an optional description